urllib3/contrib/pyopenssl.py

```diff
@@ -127,6 +129,17 @@ def extract_from_urllib3():
     util.ssl_.IS_PYOPENSSL = False
 
 
+def _validate_dependencies_met():
+    """
+    Verifies that PyOpenSSL's package-level dependencies have been met.
+    Throws `ImportError` if they are not met.
+    """
+    # Method added in `cryptography==1.1`; not available in older versions
+    from cryptography.x509.extensions import Extensions
+    if not hasattr(Extensions, "get_extension_for_class"):
+        raise ImportError("'cryptography' module missing required function (try upgrading the module?): 'cryptography.x509.extensions.Extensions.get_extension_for_class'")
```

Comment:
This is failing in the flake8 tests because it's over the 99 character line length maximum. You'll need to line wrap the error message down to something like:

```python
raise ImportError("'cryptography' module missing required function (try upgrading the "
                  "module?): 'cryptography.x509.extensions.Extensions."
                  "get_extension_for_class'")
```

It may also be worth consolidating the error to a more generic "cryptography doesn't meet minimum requirements of x.x". That would allow the ImportError to encapsulate a wider range of dependency issues.